Unigram/Unigram/Controls/PlaybackSlider.cs

```diff
@@ -137,5 +146,79 @@ private void UpdateVisualState(PointerRoutedEventArgs e)
                 VisualStateManager.GoToState(this, "Normal", true);
             }
         }
+    }
+
+    public class PlaybackSliderAutomationPeer : RangeBaseAutomationPeer, IValueProvider
+    {
+        private readonly PlaybackSlider _owner;
+
+        public PlaybackSliderAutomationPeer(PlaybackSlider owner)
+            : base(owner)
+        {
+            _owner = owner;
+        }
+
+        protected override string GetClassNameCore()
+        {
+            return "Slider";
+        }
+
+        protected override string GetNameCore()
+        {
+            return "Seek";
+        }
+
+        protected override object GetPatternCore(PatternInterface patternInterface)
+        {
+            if (patternInterface == PatternInterface.Value)
+            {
+                return this;
+            }
+
+            return base.GetPatternCore(patternInterface);
+        }
+
+        protected override AutomationControlType GetAutomationControlTypeCore()
+        {
+            return AutomationControlType.Slider;
+        }
+
+        public string Value
+        {
+            get
+            {
+                return ReadableTime((int)_owner.Value);
+            }
+        }
+
+        public void SetValue(string value)
+        {
+        }
+
+        public bool IsReadOnly
+        {
+            get
+            {
+                return true;
+            }
+        }
+
+        private static string ReadableTime(int seconds)
+        {
+            var parts = new List<string>();
+            Action<int, string> add = (val, unit) => { if (val > 0) parts.Add(val + unit); };
+            var t = TimeSpan.FromSeconds(seconds);
+
+            add(t.Days, "d");
+            add(t.Hours, "h");
+            add(t.Minutes, "m");
+            add(t.Seconds, "s");
+
+            return string.Join(" ", parts);
+        }
```

Comment:
This can be improved!
First of all there's no need to cast `double` to `int`, you can use `double` directly.
Second, I feel that it might be better to use standard `TimeSpan.ToString` overload, as in the code suggestion.

Here I'm not actually sure whether screen readers are able to read `00:05` in a decent way, tho.
If it's not the case, you can use `Locale.Declension` method to pluralize hours, minutes, and seconds (no need to include days here :)) respecting current language rules.

```suggestion
        private static string ReadableTime(double seconds)
        {
            var duration = TimeSpan.FromSeconds(seconds);
            if (duration.TotalHours >= 1)
            {
                return duration.ToString("h\\:mm\\:ss");
            }
            else
            {
                return duration.ToString("mm\\:ss");
            }
        }
```